mono/mini/mini-runtime.h

```diff
@@ -174,6 +174,18 @@ typedef struct MonoDebugOptions {
 	gboolean dyn_runtime_invoke;
 	gboolean gdb;
 	gboolean lldb;
+
+	/*
+	 * With LLVM codegen, this option will cause methods to be called indirectly through the
+	 * PLT (As they are in other FullAOT modes, without LLVM). The indirection allows 
+	 * class constructors to be executed before the first time that any method of a 
+	 * given class is called. Without this option, LLVM codegen will insert code at the
+	 * beginning of every method to check the initialization status and call the class 
+	 * constructors, if necessary. With this option, LLVM will do a slightly worse job 
+	 * of code generation, but the result will be slightly smaller. Without this 
+	 * option, LLVM may be able to optimize more aggressively, but the code may be bigger.
+	 */
```

Comment:
the option name already contains a negation, so this is hard to read imho. could you rephrase it to `if the flag is set, then [...]. if the flag isn't set then [...]`? maybe use bullet points.

Reply:
Github doesn't show this as out of date, but this one is